src/main/java/com/wise23/chariteed/service/PatientService.java

```diff
@@ -0,0 +1,16 @@
+package com.wise23.chariteed.service;
+
+import com.wise23.chariteed.ChariteEdApplication;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.client.api.IGenericClient;
+
+public class PatientService {
+    public FhirContext fhirContext;
+    public IGenericClient client;
+
+    public PatientService() {
+        this.fhirContext = FhirContext.forR4();
+        this.client = fhirContext.newRestfulGenericClient(ChariteEdApplication.server_url);
```

Comment:
This can be done in a configuration file, fhirContext is screaming to be defined as a separate bean in a config file.

Reply:
Can the example in here https://www.baeldung.com/spring-bean be used to achieve this?

Reply:
@JannesVolkens i think this will do the job